pkg/testgridanalysis/testidentification/test_identification.go

```diff
@@ -17,12 +17,14 @@ var customJobSetupContainers = sets.NewString(
 	"e2e-azure-upgrade-ipi-conf-azure",
 	"e2e-gcp-upgrade-ipi-install-install-stableinitial",
 	"e2e-metal-assisted-baremetalds-assisted-setup",
+	"e2e-metal-assisted-ipv6-baremetalds-packet-setup container test",
 	"e2e-metal-assisted-onprem-baremetalds-assisted-setup",
 	"e2e-metal-ipi-baremetalds-devscripts-setup",
 	"e2e-metal-ipi-ovn-dualstack-baremetalds-devscripts-setup",
 	"e2e-metal-ipi-ovn-ipv6-baremetalds-devscripts-setup",
 	"e2e-metal-ipi-upgrade-baremetalds-packet-setup",
 	"e2e-metal-ipi-virtualmedia-baremetalds-devscripts-setup",
+	"e2e-ovirt-ipi-install-ovirt-setup container test",
```

Comment:
i just meant you could add "setup container test" as the string here(which would cover both jobs).  but this is fine.